app/food_truck.rb

```diff
@@ -0,0 +1,26 @@
+# Retrieves food-truck feed and parses it
+
+require_relative '../constants'
+require 'httparty'
+require 'nokogiri'
+
+module FoodTruck
+  class FoodTruck
```

Comment:
IMO, it's confusing to name this `FoodTruck` given that the outer module is also `FoodTruck`.  In some contexts, it's not clear which constant you are referring to by `FoodTruck`.  For example, in your API:

``` ruby
module FoodTruck
  class API < Sinatra::Base
    get '/api' do
      ft = FoodTruck.new

      SpeakSlack.trucks(trucks: ft.trucks)
    end
  end
end
```

...it looks like `ft = FoodTruck.new` is trying to create an instance of the outer module, but of course that isn't possible, and looking at the entire code base I can see it is making a `FoodTruck::FoodTruck` instance.  But it's still confusing.

I recommend calling this something like `FoodTruck` parser.